webservices/rest.py

```diff
@@ -639,6 +648,18 @@ def get(self):
             args['year'] = default_year()
         year = args['year']
 
+        # Checking for special cases
+        if 'q' in args and args['q'] is not None:
+            # use committee id
```

Comment:
It wouldn't be hard to add committee_id and candidate_id to the fulltext search index, so they would automatically be found by the regular q= queries, and I think that would accomplish the same thing - would that be a better solution?

(sorry, I feel like I would have been able to suggest this before you did the work if I'd been paying better attention)